contrib/epee/include/net/local_ip.h

```diff
@@ -66,6 +68,20 @@ namespace epee
       */
       return false;
     }
+
+    inline
+    bool is_ipv6_local(const std::string& ip)
+    {
+      // ipv6 local addresses start with fc00
+      return (ip.find("fc00") == 0) || (ip.find("FC00") == 0);
```

Comment:
`fd00` as well

Reply:
I...huh, I believe you, but now I'm confused as to why that wasn't listed where I found the others.

Reply:
"The address block fc00::/7 is divided into two parts, fc00::/8 and fd00::/8."

oh god dammit.